README.md

```diff
@@ -1,9 +1,13 @@
+https://d1rkn6vxu8pvu6.cloudfront.net/(https://d1rkn6vxu8pvu6.cloudfront.net/)
+
+http://shop-react-redux-cloudfront-v.s3-website.eu-central-1.amazonaws.com/(http://shop-react-redux-cloudfront-v.s3-website.eu-central-1.amazonaws.com/)
```

Comment:
you can configure your s3 bucket to https, and after that, your will see on your page of app:
![image](https://user-images.githubusercontent.com/25940019/178492611-db23c861-446d-4139-b155-73901426a278.png)
it's from our task: 
![image](https://user-images.githubusercontent.com/25940019/178492849-d429324d-ee23-4eec-9cae-a97fed2f53b0.png)

Reply:
Hi, thank you for the review! Actually this 403 Error occurs when we apply a 'special' policy after creating a cloudfront distribution. So that only through cloudfront our webapp can be distributed. And as I still playing around with it currently I had this policy reverted during your check :)
Also I've added additional URLs to check in the readme file. First one URLs were for deployment without serverless-single-page-app.